packages/allure-vitest/package.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "allure-vitest",
+  "version": "2.10.0",
```

Comment:
Should it start with a beta for initial testing?

Reply:
No need, I guess because we release all the packages at once with the same version. I'll add all the missing methods to make the integration much more functional